src/modules/project-config.ts

```diff
@@ -169,6 +168,55 @@ export class ProjectConfig {
         return maybeComponentConfig ? maybeComponentConfig : new ComponentConfig(componentId);
     }
 
+    /**
+     * @param onlyInstalled only retrieves the installed packages for the project. Defaults to false.
+     * @returns all used packages by the project.
+     */
+    getPackages(onlyInstalled: boolean = false): PackageConfig[] {
+        if (onlyInstalled) {
+            return this._packages.filter((pkg) => pkg.isPackageInstalled());
+        }
+
+        return this._packages;
+    }
+
+    getPackageConfig(packageName: string, onlyInstalled: boolean = false): PackageConfig | undefined {
```

Comment:
add missing tsdocs.

onlyInstalled is not used in the logic?

Reply:
Good catch! Added the onlyInstalled parameter to the getPackages() call.